core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -56,7 +56,7 @@ class ReplicaFetcherThread(name: String,
                                 isInterruptible = false,
                                 includeLogTruncation = true) {
 
-  type REQ = FetchRequest
+  type REQ = kafka.server.ReplicaFetcherThread.FetchRequest
```

Comment:
This seems to be unrelated to this PR?

Reply:
There was some ambiguity errors in checkstyle which I could not fix without doing this when I am running tests in local. I could revert it later, just before merging.

Reply:
I am wondering how this could happen -- checkstyle runs on each build and passes. Can you share the error you get?

Reply:
Oh, going back and checking. I could not replicate that error. I would fix this.